src/components/Updates/FinancialCard.css

```diff
@@ -22,3 +32,7 @@
 .FinancialCard__Value > .Text.FinancialCard__Text {
   color: var(--white-900) !important;
 }
+
+.Link.FinancialCard::after {
```

Comment:
Is it possible to not use Link here and pass the class `FinanicalCard__LInk` to the link component directly?